app/src/main/java/com/veephealthoutloud/healthoutloud/Fragments/NewsfeedFragment.java

```diff
@@ -167,6 +180,8 @@ public void onActivityCreated(Bundle savedInstanceState) {
         feelings.add("sad");
         feelings.add("frustrated");
 
+        getAllPosts();
+
```

Comment:
Yeah I added GetPosts() as a placeholder to use before we put the requests in, so you should remove this and change your GetAllPosts() to GetPosts() so it can be used by the Newsfeed Fragment

Reply:
Also, since we're using the newsfeed to show posts for a certain feeling, you might want to add a parameter for the feeling so it can be used in the request. This can be added later though